schema.sql

```diff
@@ -30,4 +30,26 @@ CREATE TABLE species (
 
 ALTER TABLE animals DROP COLUMN species;
 ALTER TABLE animals ADD COLUMN species_id INT REFERENCES species(id); 
-ALTER TABLE animals ADD COLUMN owner_id INT REFERENCES owner(id);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN owner_id INT REFERENCES owner(id);
+
+--join table
+CREATE TABLE vets(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    age INT,
+    date_of_graduation DATE
+);
+SELECT * FROM vets;
+
+CREATE TABLE specializations (
+    species_id INT REFERENCES species(id),
+    vets_id INT REFERENCES vets(id),
+    PRIMARY KEY (species_id, vets_id)
+);
+
+CREATE TABLE visits (
+    animals_id INT REFERENCES animals(id),
+    vets_id INT REFERENCES vets(id),
+    visit_date DATE NOT NULL,
+    PRIMARY KEY (animals_id, vets_id,visit_date);
```

Comment:
- [ Required ] I think it would be better if you remove the `primary key` constraint on the `animals_id`, `vets_id`, `visit_date` this constraint would prevent an entry where an `animal` visits the same `vet` more than once on the same `date` at different times. Please consider removing it.

Reply:
@Felix45 
Should I remove all primary keys on that table?I have the primary key for three columns now so should I remove the primary key from all?